src/components/DraggableLineChart/examples/02.ExampleWithExternalXAxisRenderProp.tsx

```diff
@@ -135,7 +127,7 @@ export default createClass({
 		return (
 			<div style={style}>
 				<DraggableLineChart
-					data={_.map(customSpendDataPoints, x => x)}
```

Comment:
this was causing some really odd behavior. There was no reason to map this array.